### ISSUE TITLE
Might not recognize correct name on specific tracking

### ISSUE THREAD

**Issue by EuAmoCafeh** [Triky313/AlbionOnline-StatisticsAnalysis#134]
I already installed all the necessary applications, even so it doesn't count the collected silver!
 
 - OS: WIN 10
 - Browser CHROME

**Comment by Triky313** [comment]
@EuAmoCafeh 
Do you have any more information for me, because I can't find any errors.
What version of the tool do you have?
Have you ever switched maps before?

**Comment by EuAmoCafeh** [comment]
I'am in the last one, and i have this error in my name ! '' This character is not tracked ''; what i do ?
<img width="840" alt="Screenshot_1" src="https://user-images.githubusercontent.com/103466580/163491589-9387889f-4000-4c33-aa5d-128fd2a454de.png">

**Comment by Triky313** [comment]
Please send me your correct inGame name. I will test this name.

You should first remove your ingame name from the settings, then it should work for now.

**Comment by EuAmoCafeh** [comment]
2 L 6 i 2 l = lliiiiiill

**Comment by EuAmoCafeh** [comment]
I removed my name within the application and it started working again, for now I didn't do much content, but for the tomes and the silver bags they already counted, and apparently the error also came out!
![image](https://user-images.githubusercontent.com/103466580/163608577-a732e30d-a69b-4b82-a7e2-a68a96280cd6.png)

**Comment by EuAmoCafeh** [comment]
![image](https://user-images.githubusercontent.com/103466580/163608729-432a767e-9682-4476-bcfc-47982529bc09.png)
![image](https://user-images.githubusercontent.com/103466580/163608775-76296d63-262d-4eea-8db1-ebf83faa8c7b.png)

**Comment by Triky313** [comment]
What exactly issue still occurs after removing the name?
Do you expect points at ReSpec?

**Comment by EuAmoCafeh** [comment]
No, I commented again to explain how I solved this problem by removing my name from the app.
What I really wanted was the silver count, as I do HCE is a help for me to get a better idea of what I'm doing.
I would even like to thank you for your work, excellent application, you are a special person, I wish you the best possible things in the world, I hope all goes well with you! I wish you competence and good luck!
